### DISCUSSION TITLE
Documentation site?

### DISCUSSION THREAD

**Discussion by jacebenson** [arnoudkooi/SN-Utils#254] [Ideas]
Would you like a documentation site?  I can put together a proof of concept if you want.  All the features... there's so many, might be a great way to keep things clear what SN utils is doing.  If not just close this.  If you want this, just comment @arnoudkooi

**Comment by arnoudkooi** [comment]
Documentation is the most requested "feature", in the recent survey (2k responses)
I'll check in with you to discuss

**Comment by arnoudkooi** [comment]
Still something that needs to be done. Moving it to a discussion for now.

**Comment by Anduh** [comment]
Site with documentation exists now: https://snutils.com/docs

this could be closed

**Comment by arnoudkooi** [comment]
Thanks @Anduh and @jacebenson
